### PR TITLE
updating System.String samples for Try .NET

### PR DESCRIPTION
**Pull request by WilliamAntonRohm**
per https://github.com/dotnet/dotnet-api-docs/pull/2971

### CODE REVIEW

**Comment by WilliamAntonRohm** [comment]
@mairaw, @BillWagner -- please hold off on reviewing this PR until I resolve the conflicts.

#hold-off

**Comment by WilliamAntonRohm** [comment]
@mairaw -- Maira, feel free to review the non-conflicting samples.  Thank you.

**Comment by mairaw** [comment]
Are you going to fix the merge conflicts?

**Comment by WilliamAntonRohm** [comment]
@mairaw, @BillWagner -- please conclude your review; my apologies for the earlier merge conflicts.

File: snippets/csharp/VS_Snippets_CLR/string.compare4/CS/comp4.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿//<snippet1>
+//<snippet1>
```
Comment (BillWagner) [inline snippets/csharp/VS_Snippets_CLR/string.compare4/CS/comp4.cs]: @WilliamAntonRohm Is this one supposed to be interactive? If so, the `<snippet1>` and `</snippet1`> markers should move to inside `Main`. (I can't tell from just this PR.)

Comment (WilliamAntonRohm) [reply snippets/csharp/VS_Snippets_CLR/string.compare4/CS/comp4.cs]: @BillWagner this sample is not interactive, as it relies on culture settings.

**Comment by WilliamAntonRohm** [comment]
#sign-off

**Pull request merged**